### ISSUE TITLE
how to we can add songs which we can play too 

### ISSUE THREAD

**Issue by saurabh-sen** [CleverProgrammers/spotify-clone#27]
i built the whole spotify but now i want to add play functionality too, how i can do that ???????????????????????????????????????

**Comment by dhwanit-kapur** [comment]
Hey buddy! Did you figure that out yet ? I am curious to know that as well

**Comment by ashlex21** [comment]
Same Here man!!

**Comment by kkanekii** [comment]
It's pretty easy, you need to add a seperate library for it, then make it

**Comment by Gaurav3009** [comment]
Hi! I am also facing the same issue.. did you guys got any solution for this?

**Comment by vGupta1305** [comment]
> It's pretty easy, you need to add a seperate library for it, then make it

Which Library?

**Comment by saad-official** [comment]
yes add any other library but for spotify-web-api you need to have premium account to play a song
